torch/_dynamo/symbolic_convert.py

```diff
@@ -741,6 +739,28 @@ def COMPARE_OP(self, inst):
                     **options,
                 )
             )
+        elif (
+            isinstance(left, TensorVariable) or isinstance(right, TensorVariable)
+        ) and op in supported_tensors:
+            self.push(
+                wrap_fx_proxy(
+                    self,
+                    supported_tensors[op](left.as_proxy(), right.as_proxy()),
+                    **options,
+                )
+            )
+        elif (
+            isinstance(left, DynamicShapeVariable)
+            or isinstance(right, DynamicShapeVariable)
+        ) and op in supported_tensors:
+            self.push(
+                DynamicShapeVariable.create(
```

Comment:
Odd that wrap_fx_proxy not used here

Reply:
We could, although I like having this here as we can explicitly dictate what it should be. Similar patterns exist on DataClassVariable.create, GradModeVariable.create, etc.